Created a new method that takes two parameters at the input

YevheniiKobko opened the pull request.

YevheniiKobko committed:
Created a new method that takes two parameters at the input

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -5,6 +5,7 @@
 public class SumOfPlanes {
     public static void main(String[] args) {
         System.out.println("Your number: "+ sum());
+
```
This line should be removed

YevheniiKobko replied:
done

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -5,6 +5,7 @@
         System.out.println("Your number: "+ sum());
+
     }
     /* Creating a variable int sum. It calculates the sum of two numbers*/
```
In the single comment, we should use // instead of /*

YevheniiKobko replied:
done

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -5,6 +5,7 @@
 public class SumOfPlanes {
     public static void main(String[] args) {
         System.out.println("Your number: "+ sum());
```
After/before `+` you need to add white space

YevheniiKobko replied:
done

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -16,6 +17,11 @@ public static int sum () {
         int sum = a+b;
```
After/before + you need to add white space

YevheniiKobko replied:
done

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -16,6 +17,11 @@ public static int sum () {
         int sum = a+b;
         return sum;
     }
+    /*Created a new method that takes two parameters at the input*/
```
In the single comment, we should use // instead of /*

JiPolik replied:
Please provide comment regarding the Java official documentation.

YevheniiKobko replied:
done

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -16,6 +17,11 @@ public static int sum () {
+        int sum = firstNumbers + secondNumbers;
+        return sum;
+    }
 
```
Empty lines should be removed also

YevheniiKobko replied:
done

YevheniiKobko committed:
Minor bug fixes

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -4,18 +4,21 @@
 public class SumOfPlanes {
     public static void main(String[] args) {
-        System.out.println("Your number: "+ sum());
+        System.out.println("Your number: " + sum());
```
When do you call the method `sumOfNumber`?
You called only the method `sum`.
Try to call the method `sumOfNumber` in the `main` method and pass into this method two arguments regarding the structure of the method.

YevheniiKobko committed:
Fixed errors with call method sumOfNumbers

Created a new method that takes two parameters at the input and calculates their sum and calling it in the method main.

JiPolik commented on src/java/com/trade/delivery/SumOfPlanes.java:
```diff
@@ -1,21 +1,14 @@
+    //Created a new method that takes two parameters at the input and calculates their sum.
+    public static int sumOfNumbers (int a, int b){
+        int c = a + b;
+        return (c);
```
What was the reason add the round brackets for variable `c`?

YevheniiKobko committed:
Fixed a bug with the return value
